src/SFA.DAS.AssessorService.Web/StartupConfiguration/AuthenticationStartup.cs

```diff
@@ -52,9 +52,7 @@ public static class AuthenticationStartup
                     };
 
                     options.SignInScheme = "Cookies";
-                    options.Authority = _configuration.LoginService.MetadataAddress;
```

Comment:
Yes - here is the method with a couple of lines removed - but what else calls this?

Reply:
Nothing that I can see. So the whole class is redundant now.